collection-scripts/gather_debug

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+FILE=/tmp/rm-to-finish-gathering
+MSG=""\
+"Must gather entering debug mode, will sleep until file $FILE is deleted\n"\
+"You can go into the container now and execute /usr/bin/gather/n"
+
+echo -e $MSG
```

Comment:
-e "${MSG}"

Reply:
No need, echo command is ok like this as long as we don't start the MSG with a "-", which we won't do.